doc/api/tls.md

```diff
@@ -876,6 +876,27 @@ socket.on('end', () => {
 });
 ```
 
+When using an instance of `net.Socket`, use `net.Socket` to upgrade an existing socket. Do not wrap the `net.Socket` in a `TLSSocket`. See the example below for usage of upgrading an existing socket:
+
+```js
+const Socket = require('net').Socket;
+const tls = require('tls');
+const sock = new Socket();
+const secureSock = tls.connect({ socket: s }, () => {
+  console.log('The tls socket connected.');
+});
+sock.connect({ port: 6697, host: 'irc.freenode.net' });
```

Comment:
I replaced `{ socket: s }` with the port and host and removed `sock.connect` and it appears to be passing, and now the sample code is more in line with the docs' recommendations.